lib/helpers/visible.js

```diff
@@ -0,0 +1,6 @@
+// Visible logic based on jQuery's
+// https://github.com/jquery/jquery/blob/4a2bcc27f9c3ee24b3effac0fbe1285d1ee23cc5/src/css/hiddenVisibleSelectors.js#L11-L13
+
+export default function visible(el) {
+  return !!(el.offsetWidth || el.offsetHeight || el.getClientRects().length);
```

Comment:
could make this check a little more explicit? assuming I understand the logic correctly, maybe something like this:

```js
return el.offsetWidth !== 0 || el.offsetHeight !== 0 || el.getClientRects().length !== 0
```

Reply:
This seems like a good change, though the above is literally what jQuery uses (though that isn't any endorsement for using that code).